🇺🇦 Ukrainian localization of OpenTelemetry documentation

Andygol opened the pull request:
I'm thrilled to share the news that I've translated the OpenTelemetry.io website into Ukrainian — https://andygol-otel.netlify.app/uk/! If this is something you're interested in, you're more than welcome to merge it to the main repo.

| <img width="1387" alt="image" src="https://github.com/user-attachments/assets/ea346774-5e31-420f-9bb7-e8ed12f33696" /> |  <img width="1387" alt="image" src="https://github.com/user-attachments/assets/aa828f0c-998a-4197-b8f9-45af1ad9a8c6" /> |
| --- | --- |

Andygol committed:
[uk] Ukrainian documentation for OpenTelemetry

[uk] Community resources and feedback channels

[uk] Site information page

[uk] OpenTelemetry ecosystem

[uk] OpenTelemetry concepts

[uk] OpenTelemetry Demo

[uk] OpenTelemetry with various programming languages

[uk] OpenTelemetry with zero-code instrumentation

[uk] OpenTelemetry Collector components

[uk] OpenTelemetry Operator and related components in Kubernetes

[uk] OpenTelemetry with Functions as a Service (FaaS) and Lambda instrumentation

[uk] Migration to OpenTelemetry from OpenTracing and OpenCensus

[uk] Specifications and status of OpenTelemetry signals

[uk] Security best practices, CVEs, and incident response in OpenTelemetry

[uk] Blog post on sunsetting OpenCensus

[uk] OpenTelemtry PHP configuration, PHP SDK documentation

[uk] OpenCensus migration redirect link to include Ukrainian path

[uk] Update What is OpenTelemetry to improve clarity and consistency

[uk] OpenTelemetry overview and Getting started

[uk] Blog on OpenTelemetry in CICD

[uk] Blog post on sunsetting OpenCensus

[uk] Blog Announcements kubecon-na

svrnm commented on content/uk/_index.md:
As [suggested on slack](https://cloud-native.slack.com/archives/C076RUAGP37/p1736237461232719?thread_ts=1735302640.330329&cid=C076RUAGP37), please follow the process to establish a localization group for Ukrainian: https://opentelemetry.io/docs/contributing/localization/#new-localizations. The first language PR should only contain this page.

svrnm reviewed (changes requested):
As [suggested on slack](https://cloud-native.slack.com/archives/C076RUAGP37/p1736237461232719?thread_ts=1735302640.330329&cid=C076RUAGP37), please follow the process to establish a localization group for Ukrainian: https://opentelemetry.io/docs/contributing/localization/#new-localizations.

chalin commented:
@Andygol - please rebase and address conflicst:

> <img width="1750" height="500" alt="image" src="https://github.com/user-attachments/assets/a7ccfe7f-b031-4d4d-b613-a27632150540" />

Ping me once that is done and I'll then look at the failing check.

chalin reviewed (changes requested):
A few broad-scoping comments:

- This PR is too big, it causes the GH web interface to hang for too long. Break is up into meaningful chunks. This will also make it more manageable for content reviewers.
- Don't copy images. Hugo will pick them up from the default locale (`en`).

chalin reviewed (changes requested):
@Andygol - per https://opentelemetry.io/docs/contributing/localization/#images, do not copy images unless they have been edited. Don't edit them unless you're updating the prose.

chalin commented on layouts/_shortcodes/uk/docs/languages/instrumentation-intro.md:
This shortcode is no longer used, as are a few other obsolete ones.

chalin reviewed (changes requested):
Clean out obsolete/out-dated partials and shortcodes. See inline comment for one example.

Andygol committed:
[uk] Blog TC election results

Andygol committed:
[uk] Blog TC election results

Andygol committed:
[uk] Blog OpenTelemetry Android

Andygol committed:
[uk] Blog OpenTelemetry Android

Andygol committed:
[uk] Blog Life of an OTel maintainer

Andygol committed:
[uk] Blog Life of an OTel maintainer

Andygol committed:
[uk] Blog Demystifying auto-instrumentation

Andygol committed:
[uk] Blog Demystifying auto-instrumentation

Andygol committed:
[uk] Blog OTel Unplugged 2026

Andygol committed:
[uk] Blog OTel Unplugged 2026

Andygol committed:
[uk] Blog Sampling milestones

Andygol committed:
[uk] Blog Sampling milestones

Andygol committed:
[uk] Blog Declarative config

Andygol committed:
[uk] Blog Declarative config

Andygol committed:
[uk] Blog GC election result

Andygol committed:
[uk] Blog GC election result

Andygol committed:
[uk] Blog OBI First release

Andygol committed:
[uk] Blog OBI First release

Andygol committed:
[uk] Blog Complex attribute types

Andygol committed:
[uk] Blog Complex attribute types

Andygol committed:
[uk] Blog Unroll processor

Andygol committed:
[uk] Blog Unroll processor

Andygol committed:
[uk] Blog Stability proposal

Andygol committed:
[uk] Blog Stability proposal

Andygol committed:
[uk] Blog OTCA

Andygol committed:
[uk] Blog OTCA

Andygol committed:
[uk] Blog Deprecating Zipking exporters

Andygol committed:
[uk] Blog Deprecating Zipking exporters

Andygol committed:
[uk] Blog Demystifying OpenTelemetry

Andygol committed:
[uk] Blog Demystifying OpenTelemetry

Andygol committed:
[uk] Blog Slack community insights

Andygol committed:
[uk] Blog Slack community insights

andrew-demb commented on content/uk/docs/zero-code/java/agent/declarative-configuration.md:
```diff
@@ -0,0 +1,205 @@
+
+1. Якщо параметр конфігурації починається з `otel.javaagent.` (наприклад, `otel.javaagent.logging`), то, найімовірніше, це властивість, яку можна встановити лише за допомогою змінної середовища або системної властивості (докладнішу інформацію дивіться у розділі [Параметри, доступні лише для змінних середовища та системних властивостей](#environment-variables-and-system-properties-only-options) ни ...
+2. Якщо опція конфігурації починається з `otel.instrumentation.` (наприклад, `otel.instrumentation.spring-batch.experimental.chunk.new-trace`), видаліть префікс `otel.instrumentation.` і розмістіть її в розділі `instrumentation` нижче.
+3. В іншому випадку опція, швидше за все, належить до конфігурації SDK. Знайдіть відповідний розділ у [конфігурації міграції](https://github.com/open-telemetry/opentelemetry-configuration/blob/main/examples/sdk-migration-config.yaml). Якщо у вас є системна властивість, така як `otel.bsp.schedule.delay`, знайдіть відповідну змінну середовища `OTEL_BSP_SCHEDULE_DELAY` у конфігурації міграції.
```
```suggestion
3. В іншому випадку опція, швидше за все, належить до конфігурації SDK. Знайдіть відповідний розділ у [конфігурації міграції](https://github.com/open-telemetry/opentelemetry-configuration/blob/main/examples/otel-sdk-migration-config.yaml). Якщо у вас є системна властивість, така як `otel.bsp.schedule.delay`, знайдіть відповідну змінну середовища `OTEL_BSP_SCHEDULE_DELAY` у конфігурації міграції.
```

Dead link

Andygol committed:
[uk] Blog Node.js DDOS

Andygol committed:
[uk] Blog Node.js DDOS

Andygol committed:
[uk] Blog Log deduplicatiion processor

Andygol committed:
[uk] Blog Log deduplicatiion processor

Andygol committed:
[uk] Blog OBI Goals 2026

Andygol committed:
[uk] Blog OBI Goals 2026

Andygol committed:
[uk] Blog Dapr workflow observability

Andygol committed:
[uk] Blog Dapr workflow observability

Andygol committed:
[uk] Blog Otel collector survey follow up

Andygol committed:
[uk] Blog Otel collector survey follow up

Andygol committed:
[uk] Blog Alternative approaches to contributing

Andygol committed:
[uk] Blog Alternative approaches to contributing

Andygol committed:
[uk] Blog Devex Mastodon

Andygol committed:
[uk] Blog Devex Mastodon

Andygol committed:
[uk] Homepage update with new design

Andygol committed:
[uk] Blog Blueprints intro

Andygol committed:
[uk] Blog Blueprints intro

Andygol committed:
[uk] Blog Otel graduates

Andygol committed:
[uk] Ukrainian documentation for OpenTelemetry

[uk] Community resources and feedback channels

[uk] Site information page

[uk] OpenTelemetry ecosystem

[uk] OpenTelemetry concepts

[uk] OpenTelemetry Demo

[uk] OpenTelemetry with various programming languages

[uk] OpenTelemetry with zero-code instrumentation

[uk] OpenTelemetry Collector components

[uk] OpenTelemetry Operator and related components in Kubernetes

[uk] OpenTelemetry with Functions as a Service (FaaS) and Lambda instrumentation

[uk] Migration to OpenTelemetry from OpenTracing and OpenCensus

[uk] Specifications and status of OpenTelemetry signals

[uk] Security best practices, CVEs, and incident response in OpenTelemetry

[uk] Blog post on sunsetting OpenCensus

[uk] OpenTelemtry PHP configuration, PHP SDK documentation

[uk] OpenCensus migration redirect link to include Ukrainian path

[uk] Update What is OpenTelemetry to improve clarity and consistency

[uk] OpenTelemetry overview and Getting started

[uk] Blog on OpenTelemetry in CICD

[uk] Blog post on sunsetting OpenCensus

[uk] Blog Announcements kubecon-na

chalin commented:
👋 Please update this PR's branch over the latest `main`: it predates our link-checker [switch to Lychee][#10911] and can hit CI failures unrelated to your changes. For the how — including conflicts and after-update steps — see [#10990][]. Thanks for your contribution!

[#10911]: https://github.com/open-telemetry/opentelemetry.io/pull/10911
[#10990]: https://github.com/open-telemetry/opentelemetry.io/issues/10990

Andygol committed:
[uk] Blog Otel grad what next

Andygol committed:
[uk] Blog Go Compile-Time Instrumentation

Andygol committed:
[uk] Blog Dart Flutter

Andygol committed:
[uk] Blog Lambda in OTTL

Andygol commented:
@chalin, I’ve updated the current codebase to use Lychee and also synced the Ukrainian translation.

UPD The i18n checks are currently failing, but not because of errors in the Ukrainian files—rather, because of errors in other languages

Andygol committed:
[uk] Blog We brok the Demo

chalin committed:
Replace fork-local default_lang_commit hashes with upstream merge-base

chalin committed:
Fix stale links and update drift status for uk pages

- Update 5 link occurrences (4 pages) to the targets current en pages use
- Mark 16 uk pages with drifted_from_default: true via npm run fix:i18n:status

chalin reviewed (commented):
Hi @Andygol, I've pushed fixes to your branch. CI is now green.

**Root cause of the I18N failures**: 11 pages pinned `default_lang_commit` to a commit that exists only in your fork, not upstream, so `check-i18n.sh` couldn't `git diff` against it (the `HASH ERROR (128)` lines).

**What we changed**, in three commits on top of your work:

1. Rebased the branch over upstream `main`, resolving the `.lycheecache` conflict by taking main's copy, and replaced the fork-local `default_lang_commit` hashes with the upstream merge-base.
2. Marked 16 pages `drifted_from_default: true` via `npm run fix:i18n:status`, and updated 5 links (4 pages) whose English counterparts had been fixed since translation.
3. Committed the regenerated `.lycheecache` (7 new entries for links added by your pages), since the `CACHE updates committed?` check requires the cache to be current.

**To sync your local clone**:

```sh
git fetch origin
git reset --hard origin/main-uk
```

**Tips for next time**:

- Before pinning `default_lang_commit` values, fast-forward your local `main` to upstream `main`: the i18n tooling resolves `HEAD`/branch names against your local clone, which is how the fork-only hash got pinned.
- Prefer `npm run fix:i18n:status -- <paths>` to set or update drift state, and `npm run check:i18n` to validate locally.
